src/utils.py

```diff
@@ -0,0 +1,187 @@
+import mysql.connector as sql # from dotenv import load_dotenv
+import os
+from typing import Tuple
+from dotenv import load_dotenv, find_dotenv
+from mysql.connector import Error
+
+load_dotenv(override=True)
+
+# TODO-1: MySQL connection Method
+def connect_to_mysql(host:str='localhost', user:str='root', port:int=3306, database:str=None) -> Tuple[sql.connection.MySQLConnection, sql.connection.MySQLCursor]:
+    """ Connection to MySQL Server
```

Comment:
make sure to put args on newline so the whole code is visible on the laptop screen when reviewing